src/common/common.go

```diff
@@ -16,7 +16,11 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
-const ControllerLogsSecondsAgo = 60 * 60
+const (
+	ControllerLogsSecondsAgo       = 60 * 60
```

Comment:
I see you didn't add it in this PR, yet what is this const? can it get a better name?

Reply:
Have a better name? will be glad to change. It says bring controller logs since hour ago. Not sure what is the best name for it

Reply:
we can though drop it and just bring all logs

Reply:
next time